src/netspresso_trainer/models/op/custom.py

```diff
@@ -140,11 +140,14 @@ def __init__(
         dilation: int = 1,
         norm_layer: Optional[str] = None,
         expansion: Optional[int] = None,
-        no_relu: bool = False
+        act_type: Optional[str] = None,
+        no_out_act: bool = False
```

Comment:
@illian01 

Would it be better if we equalize to be same with ConvLayer case:

```suggestion
        use_act: bool = True
```

Reply:
@deepkyu 
I changed `no_relu` to `no_out_act`, because this determines activation only for last output.
Wouldn't be confusing if we use `use_act`?
Because `use_act` in ConvLayer determines activation of ConvLayer, while `no_relu` determines only last output.

Reply:
Got it. Just leave it as is.